components/Gallery/GalleryItems/FileViewerCard.vue

```diff
@@ -104,7 +104,7 @@ export default {
   },
   computed: {
     isReady() {
-      return this.data.title && (this.thumbnail || this.useDefaultImg) && (this.data.link || this.data.userData)
```

Comment:
why are we removing the check for the title? Images should have a title associated with them

Reply:
Due to the failed biolucida requests, the file name will be `undefined`. The main reason for removing the title check is to avoid this situation. This may cause some inconvenience for people who are doing the portal QC. I updated the [existing biolucida testing](https://github.com/ABI-Software/scicrunch-knowledge-testing/pull/18) which will run through all the datasets and help to generate a report of related issues.